ECG_Arrhythmia_Detection_workshop.ipynb

```diff
@@ -0,0 +1,609 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e80693de-5d45-4ca6-9eec-8f7a8239d9ca",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "**Before You Start**: Make sure that you ran `0_setup_workshop.ipynb`!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bdd892d1-5661-49bd-82bd-2516ca07b9a4",
+   "metadata": {},
+   "source": [
+    "# ECG Arrhythmia detection workshop\n",
+    "\n",
+    "This project aims to demonstrate the Anomaly Detection module implemented in Darts\n",
+    "### Dataset\n",
+    "The MIT-BIH Supraventricular Arrhythmia Database (SVDB) contains 2 channels, and 78 half-hour ECG recordings obtained from 47 objects between 1975-1979.\n",
+    "\n",
+    "### Task\n",
+    "Develop an anomaly detection model to identity arrhythmia in the ECG signal."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "51d49b07-9530-4118-80e6-7b53c3614950",
+   "metadata": {},
+   "source": [
+    "## Task #1\n",
+    "### Load data of a patient into a darts timeseries object"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d90be554-88fe-4b7e-856c-8afcbeb8abf1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "patient_number = \"842\""
```

Comment:
I'd leave patient 842 only, since the other 2 that were suggested: one was not in the file, the other one had no anomalies